.github/workflows/build_all.yml

```diff
@@ -65,5 +65,5 @@ jobs:
           cd workspace
           clang --verbose
           set -x
-          sof/scripts/fuzz.sh -b -- -DEXTRA_CFLAGS='-Werror' -DEXTRA_CXXFLAGS='-Werror' \
-              -DEXTRA_CONF_FILE='stub_build_all_${{ matrix.IPC }}.conf'
+          sof/scripts/fuzz.sh -b -'${{ matrix.IPC }}' -- -DEXTRA_CFLAGS='-Werror' -DEXTRA_CXXFLAGS='-Werror' \
```

Comment:
yamlint complains about this -> https://github.com/thesofproject/sof/actions/runs/10573576552/job/29293331587?pr=9409

Reply:
Fixed. Everything green now.